submodules/faxbox/faxbox.js

```diff
@@ -247,11 +273,15 @@ define(function(require) {
 					name: 'edit',
 					data: {
 						faxbox: self.faxboxNormalizedData(data.faxbox),
-						users: data.user_list
+						users: data.user_list,
+						phone_numbers: data.phone_numbers,
+						ensure_valid_caller_id: monster.config.whitelabel.ensure_valid_caller_id
```

Comment:
As previously discussed, you can get ride of the whole `ensure_valid_caller_id ` mechanism as well (here and in the template).